### DISCUSSION TITLE
Aim speed of rifles looks unreasonable in latest experiment

### DISCUSSION THREAD

**Discussion by waveyl** [CleverRaven/Cataclysm-DDA#46182] [Q&A]
All of them are similar.
For example, precise aim, In stable,SIG costs 174 moves and Barrett costs 260. While in experiment, SIG costs 255 and Barrett costs 261.
So I'm wondering it's a bug or a feature?
![2020-12-20 213214](https://user-images.githubusercontent.com/22638873/102714624-deb3ee80-430a-11eb-86d2-cb9ffd4c4998.png)
![2020-12-20 213158](https://user-images.githubusercontent.com/22638873/102714625-e07db200-430a-11eb-8f61-5c0ba58eda87.png)

**Comment by kevingranade** [comment]
What's unreasonable about it?

**Reply by LordNIl11** [reply]
I would think that large anti material rifles shoudl be much harder (take more time) to aim than battle rifles.it woudl be my logic.dont know if in reality this matter much..

**Reply by actual-nh** [reply]
Point. Some of the larger ones have built-in bipods, however.
